css/organisms.css

```
@@ -556,8 +556,7 @@
   align-items: center;
   justify-content: center;
   
-  /* Reserved space to prevent CLS */
-  min-height: 400px;
+  min-height: 0;
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)